continuous-delivery/readme.md

```diff
@@ -0,0 +1,145 @@
+> * A discussion of the method by which the application’s deploy-time and runtime configuration will be managed, and how this relates to the automated deployment process.
+> * _Description of the integration with any external systems. At what stage and how are they tested as part of a release? How does the technical operator communicate with the provider in the event of a problem?
+> * _A disaster recovery plan so that the application’s state can be recovered following a disaster. Which steps will need to be in place to restart or redeploy the application should it fail.
+> * _Production sizing and capacity planning: How much data will your live application create? How many log files or databases will you need? How much bandwidth and disk space will you need? What latency are clients expecting?
+> * How the initial deployment to production works.
+
+* How fixing defects and applying patches to the production environment will be handled.
+
+* How upgrades to the production environment will be handled, including data migration. How will upgrades be carried out to the application without destroying its state.
+
+### Application Release and Environment Promotion
+
+Your release manifestation process should take the deployable build artifact created from your commit stage and deploy them across all cloud environments, starting with your test environment.
+
+The test environment (_often called Integration_) acts as a gate to validate if your test suite completes successfully for all release candidates. This validation should always begin in a test environment while inspecting the deployed release integrated from the feature / release branch containing your code changes.
+
+#### The First Deployment
+
+The very first deployment of any application should be showcased to the customer in a production-like environment (_UAT_) to solicit feedback early. The UAT environment is used to obtain product owner sign-off acceptance to ultimately promote the release to production.
+
+#### Criteria for a production-like environment
+
+* Runs the same operating system as production.
+* Has the same software installed as production.
+* Is sized and configured the same way as production.
+* Mirrors production's networking topology.
+* Simulated production-like load tests are executed following a release to surface any latency or throughput degradation.
+
+#### Modeling your Release Pipeline
+
+It's critical to model your test and release process to establish a common understanding between the application engineers and customer stakeholders. Specifically aligning expectations for how many cloud environments need to be pre-provisioned as well as defining sign-off gate roles and responsibilities.
+
+![image](https://user-images.githubusercontent.com/7635865/89290276-426ca200-d61e-11ea-9c79-bd183dc72050.png)
```

Comment:
Is this someone elses image or one that you created? if you created it, can we add it to this repo?

Reply:
I created the images. Added all 3 images to the repo.